ocaml/xe-cli/newcli.ml

```diff
@@ -358,6 +358,8 @@ let main_loop ifd ofd permitted_filenames =
           marshal ofd (Blob End);
           Unix.close fd
         with
+        | Unix.Unix_error(Unix.EPIPE,_,_) ->
```

Comment:
Does it make sense to capture more errors than just `EPIPE` for this error message>

Reply:
Other errors apart from the `EPIPE` will be captured by the next match `| e ->`
I think other errors will have more info and captured by other match, the `EPIPE` happens when the server disconnected during the file transfer.